NotadogApi/Controllers/UsersController.cs

```diff
@@ -37,12 +37,12 @@ public UsersController(IUserService userService, IJwtTokenGenerator jwtTokenGene
         [HttpPost("login")]
         public async Task<IActionResult> Login(UserLoginCredentials credentials)
         {
-            var user = await _userService.GetOneByEmailAsync(credentials.Email);
+            var user = await _userService.GetOneByEmailAsync(credentials.Email.Trim());

```

Comment:
`var trimmedEmail = credentials.Email.Trim()` ?

Reply:
I think we don't need trim on BE.
we probably can add exception if `credentials .email.trim != credentials .email`

Reply:
I think later we will add full validation for email, name and password. Email validation will include trim checking

Reply:
Not a good idea to throw exception. if user inserts space accidentally, better to correct him. Also between BE and FE I prefer first one to implement.